inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp

```diff
@@ -390,45 +390,94 @@ void MKLDNNGraphOptimizer::FuseMultiplyAndAdd(MKLDNNGraph &graph) {
         parentNode->addOriginalInputPrecision(childNode->getOriginalInputPrecisionAtPort(1));
         parentNode->setAlgorithm(EltwiseMulAdd);
         parentNode->addOriginalLayer(childNode->getOriginalLayers());
-
         graph.DropNode(childNode);
     }
 }
 
 void MKLDNNGraphOptimizer::FuseConvolutionAndZeroPoints(MKLDNNGraph &graph) {
-//    auto& graphNodes = graph.GetNodes();
-//
-//    auto isSutableConvNode = [](MKLDNNNodePtr node) {
-//        if (node->getType() != Convolution)
-//            return false;
-//
-//        if (node->getParentEdges().size() < 2)
-//            return false;
-//
-//        auto* convLayer = dynamic_cast<ConvolutionLayer*>(node->getCnnLayer().get());
-//        if (convLayer == nullptr)
-//            THROW_IE_EXCEPTION << "Cannot get convolution layer " << node->getName();
-//
-//        return true;
-//    };
-//
-//    auto initializeInputZeroPoints = [](MKLDNNNodePtr node, MKLDNNNodePtr parent0, MKLDNNNodePtr parent1) {
+    auto& graphNodes = graph.GetNodes();
+
+    auto isSutableConvNode = [](MKLDNNNodePtr node) {
+        return node->getType() == Convolution;
+    };
+
+    auto initializeInputZeroPoints = [](MKLDNNNodePtr node, MKLDNNNodePtr parent0, MKLDNNNodePtr parent1) {
+        auto* convNode = dynamic_cast<MKLDNNConvolutionNode*>(node.get());
+        if (convNode == nullptr)
+            THROW_IE_EXCEPTION << "Cannot get convolution node " << node->getName();
+
+        int IC = node->getParentEdgesAtPort(0)[0]->getDims()[1];
+        int OC = node->getChildEdgesAtPort(0)[0]->getDims()[1];
+
+        if (parent0->getType() == Eltwise) {
+            // The plug-in doesn't support FP32 convolution with input/weights zero points.
+            // In case weights are in FP32 (or we have zero points on weights which are not supported by INT8 convolution) we cannot use
+            // INT8 implementation so we have to disable input zero points fusing as well.
+            if (parent1->getType() != Input || !parent1->isConstant() || parent1->getOriginalOutputPrecisionAtPort(0) != Precision::I8) {
+                return false;
+            }
+
+//            auto* eltwiseNode = dynamic_cast<MKLDNNEltwiseNode *>(parent0.get());
+            if (parent0->getAlgorithm() != Algorithm::EltwiseSubtract)
+                return false;
+
+            if (parent0->getParentEdges().size() != 2)
+                return false;
+
+            auto arg0 = parent0->getParentEdgesAtPort(1)[0]->getParent();
+            if (arg0->getType() == Input && arg0->isConstant()) {
+                if (arg0->getOriginalOutputPrecisionAtPort(0) != Precision::U8)
+                    return false;
+
+                if (parent0->getParentEdgesAtPort(1)[0]->getDims().size() < 2) {
+                    return false;
+                }
+
+                if (parent0->getParentEdgesAtPort(1)[0]->getDims()[1] != 1 &&
```

Comment:
lets add check that other dimensions = 1
\+ if we support ```per tensor``` case, check from 432 should be removed

Reply:
In fact per-tensor case is not supported in general. It can be supported relatively easy, but will be done as separate feature. Updated condition.